src/common/auth.js

```diff
@@ -0,0 +1,9 @@
+import data from './axios-config';
+
+export const refreshAccessToken = async () => {
+  const {
```

Comment:
You should always add try catch blocks whenever you are dealing with potentially error-prone things like network requests